src/style/app.module.css

```diff
@@ -520,18 +533,60 @@
   border: none;
 }
 
+.actionsButton {
+  background-color: var(--search-button-bg);
+  color: var(--brown-color);
+  border: 1px solid var(--search-button-bg) !important;
+}
+
+.actionsButton:hover {
+  box-shadow: 2.5px 2.5px 2.5px rgba(0, 0, 0, 0.3);
+  background-color: var(--search-button-bg) !important;
+  color: var(--brown-color) !important;
+  border: 1px solid var(--search-button-bg) !important;
+}
+
+.eventManagementBtn {
+  color: var(--gray-white);
+  background-color: var(--white-color) !important;
+  border-color: var(--input-shadow) !important;
+}
+
+.eventManagementBtn:hover {
+  color: var(--gray-white) !important;
+  border-color: var(--input-shadow);
+}
+
+.eventManagementSelectedBtn {
+  color: var(--brown-color);
+  background-color: var(--grey-light) !important;
+  border: 1px solid var(--gray-white) !important;
+}
+
+.eventManagementSelectedBtn:hover {
+  color: var(--brown-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`